lib/membrane_framerate_converter.ex

```diff
@@ -0,0 +1,135 @@
+defmodule Membrane.FramerateConverter do
+  @moduledoc """
+  Element converts video to target constant frame rate, by dropping and duplicating frames as necessary
+  Input video may have constant or variable frame rate.
+  Element expects each frame to be received in separate buffer.
+  Additionally, presentation timestamps must be passed in each buffer's metadata.
+  """
+  use Membrane.Filter
+  use Bunch
```

Comment:
nit: there should be a blank line before `use` and modules should be ordered alphabetically
```suggestion
  """
  
  use Bunch
  use Membrane.Filter
```